pkg/kosmosctl/join/join.go

```diff
@@ -0,0 +1,218 @@
+package join
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"github.com/kosmos.io/kosmos/pkg/kosmosctl/manifest"
```

Comment:
Pay attention to specifications：
system package
blank line
Third party package
blank line
The package of the current project